src/main/java/uk.gov.hmcts.reform.fpl.ccddatamigration/domain/common/Party.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.ccddatamigration.domain.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class Party {
+    private final String partyID;
+    private final String idamID;
```

Comment:
We will never have it, is there point adding it?

Reply:
assuming we're building this for other services that may use this. Yes, our models should be the correct structure no? I guess this could cause issues when trying to put the data back into CCD however 🤔